packages/core-data/src/selectors.ts

```diff
@@ -257,6 +258,8 @@ export function getEntity( state: State, kind: string, name: string ): any {
 	} );
 	return getEntityConfig( state, kind, name );
 }
+getEntity.__unstableNormalizeArgs =
```

Comment:
I'd be interested to see support for this for actions too. Might make sense to work on it before finishing this PR.